### PR TITLE
Upgraded to expo 51 sdk

### PR DESCRIPTION
**Pull request by It-samuel**

### CODE REVIEW

**Comment by Bhaumik-Tandan** [comment]
Please check the app throughly 
After the upgrade please check for any potential issue
specially on the navigator side

File: package-lock.json
Comment (Bhaumik-Tandan) [inline package-lock.json:1]: please remove this file since we have yarn.lock

File: yarn.lock
Comment (Bhaumik-Tandan) [inline yarn.lock:1]: we are going to use yarn.lock

File: package-lock.json
Comment (It-samuel) [reply package-lock.json:1]: Which file should I remove please

**Pull request closed without merging**